kafka/tests/test.bats

```diff
@@ -0,0 +1,11 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
+@test "Service is running" {
+  echo -e "$(hab svc status)"
+  [ "$(hab svc status | grep "kafka\.default" | awk '{print $4}' | grep up)" ]
+}
+
+@test "Listening on port 9092" {
+  result="$(netstat -peanut | grep java | grep LISTEN | awk '{print $4}' | awk -F':' '{print $2}' | grep 9092)"
+  [ "${result}" -eq 9092 ]
```

Comment:
This test failing in CI highlights that the test on line 10 will never fail. If `grep 9092` or another part of this command chain fails then the test bombs out at line 9 and doesn't reach the test.
I think this isn't too bad because the test still works when it should and fails when it shouldn't.